guides/v2.1/cloud/project/project-conf-files_services-elastic.md

```diff
@@ -20,7 +20,7 @@ functional_areas:
 *   Elasticsearch performs quick and advanced searches on products in the product catalog
 *   Elasticsearch analyzers support multiple languages
```

Comment:
@keharper But which is it? Does it analyze or support? Or are there analyzing entities which support? so tricky...

Reply:
My bad. ES analyzers are a thing. Ignore my original comment.

Reply:
so Elasticsearch analyzers are a "thing":  https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-analyzers.html -- I think add cap to avoid confusion:  Elasticsearch Analyzers support.....